tests/x11/smt.pm

```diff
@@ -57,6 +57,10 @@ sub run {
 
     assert_script_run 'smt-sync', 600;
     assert_script_run 'smt-repos';
+
+    # mirror and sync a base repo from SCC
+    assert_script_run 'smt-repos --enable-mirror SLES12-SP3-Installer-Updates sle-12-x86_64';
```

Comment:
I guess we should assume that test can be executed on any architecture. So I suggest that we use VERSION and ARCH here. We can skip sle 15 though, as RMT will replace SMT there.

Reply:
smt pattern was only avaliable on x86_64 in SLE12 that means this test could be executed only on x86_64. I suggest we don't need to pay more attention on this since RMT test on SLE15 would be more important.

Reply:
@mitio Absolutely. With SLE15 we will have separate test suite anyway. Thanks for clarification.

Reply:
@rwx788 I think you meant @mitiao instead of @mitio :)

Reply:
@mitio True, sorry for that, tab suggested wrong person and I haven't recognized it =)